database/src/migrations/20240119000000_taxonomy_tables.ts

```diff
@@ -0,0 +1,166 @@
+    COMMENT ON COLUMN taxon.itis_tsn                    IS 'ITIS primary key identifier, populated from ITIS response. ITIS (Integrated Taxonomic Information System), TSN (Taxonomic Serial Number). https://itis.gov/pdf/faq_itis_tsn.pdf';
+    COMMENT ON COLUMN taxon.bc_taxon_code               IS 'British Columbia standard taxon identifier.';
+    COMMENT ON COLUMN taxon.itis_scientific_name        IS 'ITIS taxon scientific name, populated from ITIS response.';
+    COMMENT ON COLUMN taxon.common_name                 IS 'Taxon common name, initially populated from ITIS response.';
+    COMMENT ON COLUMN taxon.itis_data                   IS 'Raw ITIS payload, populated from ITIS response.';
+    COMMENT ON COLUMN taxon.itis_update_date            IS 'The datetime the ITIS taxon was updated, populated from ITIS response.';
+    COMMENT ON COLUMN taxon.record_effective_date       IS 'Record level effective date.';
+    COMMENT ON COLUMN taxon.record_end_date             IS 'Record level end date.';
+    COMMENT ON COLUMN taxon.create_date                 IS 'The datetime the record was created.';
+    COMMENT ON COLUMN taxon.create_user                 IS 'The id of the user who created the record as identified in the system user table.';
+    COMMENT ON COLUMN taxon.update_date                 IS 'The datetime the record was updated.';
+    COMMENT ON COLUMN taxon.update_user                 IS 'The id of the user who updated the record as identified in the system user table.';
+    COMMENT ON COLUMN taxon.revision_count              IS 'Revision count used for concurrency control.';
+    COMMENT ON TABLE  taxon                             IS 'Taxon cache table, extending ITIS webservice response.';
+
+    CREATE TABLE taxon_alias (
+      taxon_alias_id          integer                   GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+      taxon_id                integer                   NOT NULL,
+      language_id             integer                   NOT NULL,
+      taxon_alias_origin_id   integer                   NOT NULL,
+      alias                   varchar(300)              NOT NULL,
+      record_effective_date   date                      DEFAULT now() NOT NULL,
+      record_end_date         date,
+      create_date             timestamptz(6)            DEFAULT now() NOT NULL,
+      create_user             integer                   NOT NULL,
+      update_date             timestamptz(6),
+      update_user             integer,
+      revision_count          integer                   DEFAULT 0 NOT NULL,
+
+      CONSTRAINT              taxon_alias_pk            PRIMARY KEY (taxon_alias_id)
+    );
+
+    COMMENT ON COLUMN taxon_alias.taxon_alias_id        IS 'System generated surrogate primary key identifier.';
+    COMMENT ON COLUMN taxon_alias.taxon_id              IS 'A foreign key that points to a taxon.';
+    COMMENT ON COLUMN taxon_alias.alias                 IS 'A taxon alias.';
```

Comment:
Just missing comments for the new language/origin fields. Otherwise looks great.